common/onepanel/base/namespace.yaml

```diff
@@ -1,4 +1,6 @@
 apiVersion: v1
 kind: Namespace
 metadata:
```

Comment:
@aleksandrmelnikov is this replacing onepanel namespace? It shouldn’t. Argo and our API/UI go into `onepanel` namespace.

Reply:
The default namespace I mentioned is the user namespace where they will execute workflows.